### ISSUE TITLE
Save doesn't Redeploy per the docs on wwwroot nor cshtml files

### ISSUE THREAD

**Issue by JohnGalt1717** [microsoft/DockerTools#123]
https://docs.microsoft.com/en-us/azure/vs-azure-tools-docker-edit-and-refresh

According to that a simple save of a file should cause docker to redeploy and then hitting refresh in the browser should just work with a aspnet core site.

But it doesn't. Nothing happens until you stop and start again.

(In general the docker tools are very poorly documented, don't work out of the box and have poor integration with visual studio (i.e. you have to manually set the startup project to the docker project, because it's not possible to select it from the startup projects drop down which wastes a TON of time.)

**Comment by haniamr** [comment]
@JohnGalt1717 thanks for reporting. If you're still facing this issue, can you please share your VS version?
For the startup project issue, the dc-proj should be automatically set as the startup project after adding docker support, this isn't the case in your environment?

**Comment by chrisdpratt** [comment]
@JohnGalt1717 I ran into a similar issue when I was playing around and switched the project dropdown to a specific project name and ran it under IIS Express. Apparently, doing this switches the startup project to that project, which then left me with no option to get back to `docker-compose`. Eventually, I found that setting the `docker-compose` project back as the Startup Project manually restored the project dropdown item and allowed me to debug again using Docker.
